docs/drivers/digital-ocean.md

```diff
@@ -30,6 +30,7 @@ Options:
 -   `--digitalocean-ssh-user`: SSH username.
 -   `--digitalocean-ssh-port`: SSH port.
 -   `--digitalocean-ssh-key-fingerprint`: Use an existing SSH key instead of creating a new one, see [SSH keys](https://developers.digitalocean.com/documentation/v2/#ssh-keys).
+-   `--digitalocean-ssh-key-path`: Local path of the Digital Ocean public key. `--digitalocean-ssh-key-fingerprint` must be provided with this option.
```

Comment:
@nathanleclaire meant the private key. Will fix this on the commit to `docker/docker.github.io`.